### DISCUSSION TITLE
Collegamento app cellulare HA

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#2148] [HUB: Home Assistant]
Ciao a tutti.
Mi servirebbe un aiuto su un problema che ho avuto oggi pomeriggio.
Ho installato l'ultimo aggiornamento 2022.12.0,
Dopo il riavvio il Mio Utente "Enrico" non era più legato al device tracker "iphone di Enrico" cioè al mio cellulare e non lo trovava nemmeno.
Ho tentato di disinstallare l'app e reinstallarla ma al momento di dare il nome al mio Iphone l'app di chiede di rinominare il device perchè HA ha gia un Device tracher con lo stesso nome. Ora mi ritrovo con due device tracker con nomi diversi ma solo uno dei due ha l'icona dell Cell l'altro no.
![image](https://user-images.githubusercontent.com/72693930/206542862-2c7126b0-e4c1-4bb1-82e2-fc269b77dbc9.png)

![image](https://user-images.githubusercontent.com/72693930/206542238-36b29331-951b-4541-97e4-1b529c773b83.png)

Qualcuno mi potrebbe aiutare per favore.
Grazie mille a tutti

**Comment by Scialla** [comment]
Dovevi andare (dall'app) in impostazioni, app complementare, cliccare sulla prima voce (casa) e su nome del dispositivo, e rimetterci il nome vecchio. 
Una volta fatto vai su impostazioni, dispositivi e servizi, app x dispositivi mobili, e cancelli quello nuovo appena creato

**Reply by Moxser** [reply]
Ciao Scialla. 
Grazie per la risposta. 
Ho impostato il nome sulla app come da vecchio dispositivo ma ora mi vede come attivo il nuovo e il vecchio non più (È sempre lo stesso cell). A questo punto posso cancellare il device vecchio e tenere il nuovo? Le automazioni le ho già sistemate e le notifiche arrivano correttamente.
